.github/workflows/build.yml

```diff
@@ -95,7 +95,7 @@ jobs:
           bake-target: cni-ipam-tags
       - name: Set platform env for pr build
         run: |
-          echo "TARGET_PLATFORMS=linux/amd64" >> $GITHUB_ENV
+          echo "TARGET_PLATFORMS=linux/amd64,linux/arm64,linux/arm" >> $GITHUB_ENV
```

Comment:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#defining-environment-variables-for-a-single-workflow
Can we define env instead?

Reply:
Sure. I will open another PR to address this. The e2e test pass rate is very low right now.